app/javascript/packages/polyfill/package.json

```diff
@@ -3,6 +3,7 @@
   "private": true,
   "version": "1.0.0",
   "dependencies": {
+    "classlist-polyfill": "^1.2.0",
```

Comment:
We currently also use [`classlist.js`](https://www.npmjs.com/package/classlist.js) elsewhere. I chose `classlist-polyfill` since it's [what USWDS uses](https://github.com/uswds/uswds/blob/45296db675672cd22171d4dcb7f663df023bc5fb/src/js/polyfills/index.js#L1-L2). We can and should update other use of `classlist.js` so we're not sending duplicate implementations.